packages/widgets/tests/src/menubar.spec.ts

```diff
@@ -760,6 +780,45 @@ describe('@lumino/widgets', () => {
           expect(cancelled).to.equal(true);
         });
       });
+
+      context('focus', () => {
+        it('should focus the first menu item on keyboard focus', () => {
+          let bar = createUnfocusedMenuBar();
+          bar.contentNode.focus();
+          expect(
+            bar.contentNode.contains(document.activeElement) &&
+              bar.contentNode !== document.activeElement
+          ).to.equal(true);
+          bar.dispose();
+        });
+
+        it('should loose focus on tab key', () => {
+          let bar = createUnfocusedMenuBar();
+          bar.contentNode.focus();
+          expect(
+            bar.contentNode.contains(document.activeElement) &&
+              bar.contentNode !== document.activeElement
+          ).to.equal(true);
+          let event = new KeyboardEvent('keydown', { keyCode: 9, bubbles });
+          bar.contentNode.dispatchEvent(event);
+          expect(bar.activeIndex).to.equal(-1);
+          bar.dispose();
+        });
+
+        it('should loose focus on shift-tab key', () => {
+          let bar = createUnfocusedMenuBar();
+          bar.contentNode.focus();
+          expect(
+            bar.contentNode.contains(document.activeElement) &&
+              bar.contentNode !== document.activeElement
+          ).to.equal(true);
+          let event = new KeyboardEvent('keydown', { keyCode: 9 });
```

Comment:
Did you forget the shift modifier?

Reply:
Yes. I was fiddling with getting the tab behaviour to work in the test and in the process of that removed too much :-).